Gemeente-DenHaag/denhaag-component-library
# Button design implementation

This pull request implements issue #54 Design Button

## What's new
- Applied button design 
- Added button design tokens to `globals.module.css` (with the `:root` selector)
- Button now supports icons on the left and right side (this uses the underlying Material UI icon support in buttons)
- A few new story templates to display the different button sizes and buttons with icons

## How to test
Run storybook locally (`yarn storybook`) and check out the button page to see the new design.
